### PR TITLE
docs(storybook): updated event names to improve DX

### LINKED ISSUE #1958: Fix event handlers' names in Storybook
# Bug Report

## eBayUI Storybook
https://opensource.ebay.com/ebayui-core/?path=/story/welcome--page

## Description
There are two options in Marko.js to name an event handler, `on-click` or `onClick`. But once the name consists of more than two words it is no longer possible to use camelCase and the options change to `onButton-click` or `on-button-click`.
Every 3 and more word event handler name in the Storybook is fully in camelCase, e.g. `onInputChange` on the `<ebay-textbox>` component.
Please change the names to avoid confusion.

### PR DESCRIPTION
**Pull request by saiponnada**
## Description

Updated event handler names to improve consistency and DX across storybook stories.
- Two-word events are refactored to use camel case e.g `onClick`
- More than two-word events are refactored to use hyphen after first two words e.g `onInput-change`
- I refactored all the stories except `ebay-video`

## References

Fixes #1958

### CODE REVIEW

**Review by LuLaValva — APPROVED** [review]
Looks awesome! Thanks for doing this.

**Review by agliga — APPROVED** [review]
Lets wait for the storybook 7 pr to be merged first before merging this.

**Comment by saiponnada** [comment]
Added file `calendar.stories.js` that got accidentally deleted from PR #2010

**Pull request merged**
